chess/src/test/java/pieces/BoardTest.java

```diff
@@ -34,4 +33,62 @@ public void create() throws Exception {
                         appendNewLine("rnbqkbnr"),
                 board.showBoard());
     }
+
+    @Test
+    @DisplayName("피스의 종류를 센다")
```

Comment:
```suggestion
    @DisplayName("32개의 피스가 모두 정상적으로 존재해야 한다.")
```

귀찮아도 테스트 코드도 더 길게 써 주면 좋겠죠?